src/module.ts

```diff
@@ -62,7 +62,7 @@ export default defineNuxtModule<ModuleOptions>({
     } else {
       nuxt.options.publicRuntimeConfig.umami = resolvedOptions as any;
     }
-
-    addPlugin({ src: resolve(runtimeDir, 'plugin'), mode: 'client' });
+    //
+    addPlugin({ src: resolve(runtimeDir, 'plugin'), mode: 'all' });
```

Comment:
Tip: you don't need `mode` optin now (unless prefer it to be super explicit, it is inferred from filename and default is all)